lib/model/model.go

```diff
@@ -231,18 +231,9 @@ func (m *Model) startFolderLocked(folder string) config.FolderType {
 
 		// Directory permission bits. Will be filtered down to something
 		// sane by umask on Unixes.
-		permBits := os.FileMode(0777)
-		if runtime.GOOS == "windows" {
-			// Windows has no umask so we must chose a safer set of bits to
-			// begin with.
-			permBits = 0700
-		}
 
-		if _, err := os.Stat(cfg.Path()); os.IsNotExist(err) {
-			if err := osutil.MkdirAll(cfg.Path(), permBits); err != nil {
-				l.Warnln("Creating folder:", err)
-			}
-		}
+		createRootFolder(cfg)
```

Comment:
I've moved off the real computer for the evening so I'm not gong to test tonight, but I can bitch about the naming :D it's a folder root, not a root folder. ;)